### DISCUSSION TITLE
part-ii/2010-150/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#348] [Question comments]
Part II 2010 Topics in Analysis: Paper 2, Section II, 11F 

https://questions.tripos.org/part-ii/2010-150/

**Comment by DannyDuberstein** [comment]
(iii) Suppose w witnesses the contrary. G(D) is closed so there's a whole ball around w not in G(D). So can continuously deform image into C. Then composing this with a rotation by 180 degrees gives a continuous function D to (C in ) D with no fixed point. 

(iv) Special case of (iii), so no.
